bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -33,7 +33,7 @@ class Neuron(CMakePackage):
     version("develop", branch="master")
     version("9.0.a2", commit="89f7dab")
     version("9.0.a1", commit="b3c4b4f")
```

Comment:
@iomaganaris should we also drop these?
```suggestion
```

Reply:
`9.0.x` are based on `master` branch so I think we shouldn't